common/rust/cctrusted_base/src/tcg.rs

```diff
@@ -0,0 +1,41 @@
+use std::collections::HashMap;
+
+pub const TPM_ALG_ERROR: u8 = 0x0;
+pub const TPM_ALG_RSA: u8 = 0x1;
+pub const TPM_ALG_TDES: u8 = 0x3;
```

Comment:
Why we need TDES? It should be deprecated. If there is usage, we must use AES.

Reply:
Yup, AES SHA1 and SHA386 are used

Reply:
updated